app/controllers/chargeback_controller.rb

```diff
@@ -728,6 +728,10 @@ def get_cis_all
       EmsCluster.all.each do |cl|
         @edit[:cb_assign][:cis][cl.id] = cl.name
       end
+    elsif @edit[:new][:cbshow_typ] == "tenants"
+      Tenant.all.each do |cl|
+        @edit[:cb_assign][:cis][cl.id] = cl.name
+      end
```

Comment:
OK, thanks, they look identical.  Please rebase and work on this PR after #7202 is merged.